scripts/set-more-info-link.py

```diff
@@ -62,85 +61,127 @@ def set_link(file, link):
+    with open(file, "w") as f:
         f.writelines(lines)
 
     return status
 
 
+manurl = "https://manned.org/"
+
+
 def main():
     parser = argparse.ArgumentParser(
-        description='Sets the "More information" link for all translations of a page')
-    parser.add_argument('-p', '--page', type=str, required=True,
-                        help='page name in the format "platform/command.md"')
-    parser.add_argument('-s', '--stage', action='store_true', default=False,
-                        help='stage modified pages (requires `git` to be on $PATH and TLDR_ROOT to be a Git repository)')
-    parser.add_argument('link', type=str)
+        description='Sets the "More information" link for all translations of a page'
+    )
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument(
+        "-c",
+        "--clip",
+        action="store_true",
+        dest="clip",
+        default=False,
+        help="Uses what you copied as URL",
+    )
+    group.add_argument(
+        "-m",
+        "--man",
+        action="store_true",
+        dest="man",
+        default=False,
+        help="Automatically add the manpage for the package as the URL",
+    )
+    parser.add_argument(
+        "-p",
+        "--page",
+        type=str,
+        required=True,
+        help='page name in the format "platform/command.md"',
+    )
+    parser.add_argument(
+        "-s",
+        "--stage",
+        action="store_true",
+        default=False,
+        help="stage modified pages (requires `git` to be on $PATH and TLDR_ROOT to be a Git repository)",
+    )
+    parser.add_argument("link", type=str, nargs="?")
     args = parser.parse_args()
 
     root = get_tldr_root()
-    pages_dirs = [d for d in os.listdir(root) if d.startswith('pages')]
+    pages_dirs = [d for d in os.listdir(root) if d.startswith("pages")]
 
     target_paths = []
     rel_paths = []
 
-    if not args.page.lower().endswith('.md'):
-        args.page = f'{args.page}.md'
+    if not args.page.lower().endswith(".md"):
+        args.page = f"{args.page}.md"
 
     for pages_dir in pages_dirs:
         pages_dir_path = os.path.join(root, pages_dir)
-        platforms = [i for i in os.listdir(
-            pages_dir_path) if i not in IGNORE_FILES]
+        platforms = [i for i in os.listdir(pages_dir_path) if i not in IGNORE_FILES]
         for platform in platforms:
             platform_path = os.path.join(pages_dir_path, platform)
             pages = os.listdir(platform_path)
-            commands = [
-                f'{platform}/{p}' for p in pages if p not in IGNORE_FILES]
+            commands = [f"{platform}/{p}" for p in pages if p not in IGNORE_FILES]
             if args.page in commands:
                 path = os.path.join(pages_dir_path, args.page)
                 target_paths.append(path)
 
     target_paths.sort()
+    if args.man:
+        page_name = os.path.basename(args.page).rstrip(".md")
+        link = manurl + page_name
```

Comment:
```suggestion
        link = manurl + page_name
        request = requests.get(link)
        if not request.status_code == 200:
            print("\x1b[31m" + link + " does not exist")
            sys.exit(1)
```

Reply:
Can use `raise SystemExit("\x1b[31m" + link + " does not exist")` instead of the combination of `print` and `sys.exit(1)`. This has the added benefit that the output will go to stderr instead of stdout (though you could do `file=sys.stderr` in the print).

Reply:
Sounds good, I can't make a suggestion on the other one, so it's @Stig's turn to apply that if they want.

Reply:
Python 3.5.0 was released 09/13/2015 and reached end-of-life in 09/20/2020. Assuming you're not running a very old LTS release (e.g. 16.04), you should be running minimally Python 3.6+.

Reply:
I think you want @Stig124

Reply:
Wrong one!